Tribler/Test/GUI/widgets/test_trustdisplaypage.py

```diff
@@ -0,0 +1,72 @@
+"""
+Test package for the TrustDisplayPage functionality, in particular the converting of data.
```

Comment:
Please create a new test package `trustdisplay` in `GUI/widgets` and place this file in it, while changing the name to something describing "the converting of data".